setup.py

```diff
@@ -66,6 +65,8 @@
     "factory-boy>=3.2",
     # For running tests in parallel
     "tblib>=2.0,<3.0",
+    # For generating timezone translations.
+    "polib>=1.2,<2.0",
```

Comment:
we have polib on line 64 - https://github.com/wagtail/wagtail/blob/main/setup.py#L64

Reply:
Good catch, thanks!